runtime/bindings/python/iree/runtime/scripts/iree_tracy_capture/__main__.py

```diff
@@ -10,9 +10,16 @@
 
 
 def main(args=None):
+    try:
+        from iree import _runtime_libs_tracy
+    except ModuleNotFoundError as e:
+        raise ModuleNotFoundError(
+            "This command requires that a tracy runtime is available "
+            "but it was not built for this platform."
+        ) from e
```

Comment:
Thanks for keeping this optional (the convenience is helpful for Linux and macOS users, but I'd prefer for Windows users to install the official Tracy tools)

Reply:
+1